lib/solid.ts

```diff
@@ -18,6 +18,15 @@ export type UmaPodConfig = {
   applicationIdToken: string;
 };
 
+export type AccessGrantFilter = {
```

Comment:
Mss beetje documentatie bij de velden hiervan? VSCode zou da dan zo mooi on hover tonen
Bijvoorbeeld niet zo duidelijk wat isProvidedTo betekent

Reply:
ForPersonalData ook niet echt duidelijk wat dat inhoudt

Reply:
Klopt da dat forPersonalData string | string[] is?

Reply:
volgens de specs kan dat allebei. Zelf wel fan van gewoon lists :)